devtools/mog/main.go

```diff
@@ -76,14 +76,14 @@ func runMog(opts options) error {
 	if len(cfg.Structs) == 0 {
 		return fmt.Errorf("no source structs found in %v", opts.source)
 	}
+
 	targets, err := loadTargetStructs(targetPackages(cfg.Structs))
 	if err != nil {
 		return fmt.Errorf("failed to load targets: %w", err)
 	}
 
 	cfg.Structs = applyAutoConvertFunctions(cfg.Structs)
 
-	log.Printf("Generating code for %d structs", len(cfg.Structs))
```

Comment:
I found this logging helpful. Could we restore this or does it not work anymore for some reason?

Reply:
Restored